CHANGELOG.md

```diff
@@ -4,10 +4,9 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Distributor: add `-distributor.otel-label-name-underscore-sanitization` and `-distributor.otel-label-name-preserve-underscores` that control sanitization of underscores during OTLP translation. #13133
```

Comment:
Sorry for late comment, but this should be a `[FEATURE]`, since it's not a breaking change by any means.